packages/schemas/src/consts/oidc.ts

```diff
@@ -41,6 +41,11 @@ export enum ExtraParamsKey {
    * organization ID.
    */
   OrganizationId = 'organization_id',
+  /**
+   * Provides a hint about the login identifier the user might use.
+   * This can be used to pre-fill the identifier field **on the first screen** of the sign-in/sign-up flow.
```

Comment:
the comment should be more specific since it only applies to the first screen

Reply:
Updated.